examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -41,11 +41,11 @@ declare_args() {
   chip_default_wifi_psk = ""
 
   # Enable TestEventTrigger in GeneralDiagnostics cluster
-  silabs_test_event_trigger_enabled = false
+  sl_enable_test_event_trigger = false
```

Comment:
If this is build-configured, it could lead to the device appearing in the wild with the feature enabled in firmware. Consider making this configurable in factory partition/settings if possible.

Reply:
Unless i misunderstood the test event triger use, isn't that the comprimise for the test event trigger? If it is required for certification, it ends up being dead code in release since you can't recompile your app.

At release, since the key won't be present in nvm3, a client won't be able to trigger test behavior.

Reply:
Just to make sure i understand, ` sl_enable_test_event_trigger` and `SL_MATTER_TEST_EVENT_TRIGGER_ENABLED` should just not be present in the code. In other words, test event trigger delagate is always present and wether handlers are registers and an enableKey provide be application specific.

Reply:
Synced on slack - resolving the comment